docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -54,6 +54,9 @@ Grafana enables you to share dashboards and panels with other users within an or
 - A Snapshot
 - An embedded link (for panels only)
 - An export link (for dashboards only)
+- A report
+
```

Comment:
Is this list just for the new share button, not inclusive of the new export button?

Reply:
For now, yeah. That may change, but I want to see how the panel sharing looks before I finalize that intro text.